src/navbar/navbar-items.tsx

```diff
@@ -119,8 +121,9 @@ export const NavbarItems = <T,>({
                         selectedIndex >= 0 &&
                         selectedIndex === index &&
                         showSubMenu;
+                    const alignLeft = index === 0 && hideNavBranding;
```

Comment:
Since we are already restricting only the first child to have the align left, this check is redundant? On the same note, then maybe can have a generic prop called `$hiddenBranding` or something